_posts/volume/2024-03-01-create-a-volume.md

```diff
@@ -0,0 +1,236 @@
+menu and click on `Manage Attachments`
+
+**Step 6**  
+Select the instance you want to attach the volume to and click on
+`Attach Volume` to attach the volume to the instance.
+
+---
+
+## Using the OpenStack CLI
+Creating a volume through the OpenStack CLI can be a bit more complicated than
+using the OpenStack Dashboard, but when you get the hang of it, it can be a
+powerful tool.
+
+**Step 1**  
+First make sure you have setup the OpenStack CLI and that you are able to
+execute commands using the `openstack` command. For more information please
+refer to the
+[Using the OpenStack CLI article](
+    {{ '/articles/using-the-cli-linux' | relative_url }}
+).
+
+**Step 2**  
+Run the following command to create a volume:
+
+```bash
+openstack volume create --size <volume_size --type <volume_type> --availability-zone <availability_zone> <volume_name>
+```
+
+**Step 3**  
+Check the status of the volume by running the following command:
+
+```bash
+openstack volume show <volume_name>
+```
+
+**Step 4**  
+Attach the volume to an instance by running the following command:
+
+```bash
+openstack server add volume <instance_name/instance_id> <volume_name/volume_id>
+```
+
+---
+
+## Mounting the volume within the instance
+After you have created and attached the volume to an instance, you can mount
+the volume within the instance. The process of mounting a volume is different
+for each operating system.
+
+Be default the volume is just empty diskspace which you can use as you like.
+The following steps will guide you through the process of mounting the volume
+an adding an filesystem to it so you can use it to store your data.
+
+[Instruction for Linux](#linux)
+
+[Instruction for Windows](#windows)
+
+### Linux
+**Step 1**  
+You can identify the volume by its size and mountpoint. For example, if the
+volume is 10GB and has no mountpoint, it is likely the volume you just
+created. it will probably look simular to `/dev/sdb` but it may be different.
+
+List the available disks on the instance by running the following command:
+
+```bash
+lsblk
+```
+
+> **Note**: The mountpoint displayed in OpenStack might not be the same as the
+mountpoint in the instance.
+
+**Step 2**  
+Create a new partition on the volume by running the following command:
+
+```bash
+sudo fdisk <volume>
+```
+
+**Step 3**  
+Format the partition by running the following command:
+
+```bash
+sudo mkfs.ext4 <volume>
+```
+
+> **Note**: You can choose any filesystem to format the partition. In this
+example, we are using `ext4`.
+
+**Step 4**  
+Create a new directory to mount the volume by running the following command:
+
+```bash
+sudo mkdir /mnt/data-volume
+```
+
+> **Note**: You can choose any directory to mount the volume. In this example,
+we are using `/mnt/data-volume`.
+
+**Step 5**  
+Mount the volume by running the following command:
+
+```bash
+sudo mount <volume> /mnt/data-volume
+```
+
+You can now use the volume to store your data.
```

Comment:
Maybe refer to https://cloudtutorials.eu/articles/identify-volumes as well here?

Reply:
added